PRD.md

```diff
@@ -0,0 +1,23 @@
+# Food-Near-My-Bus
+
+## Purpose
+
+While on the bus (or your choice of transit) in Philadelphia, you get hungry. You decide to grab a quick meal or snack near the place you want to get off at. This app lets you see cafes or restaurants, by category, in a walking distance from the bus stop you want to alight at. 
+
+## Features
+
+This is a mobile friendly app that lets you click on a transit stop in Philadelphia, and see eateries within a walking distance. The points will be filtered by transit stops (bus, regional rail, trolley etc) and type of establishment (fast food/restaurant/coffee shop, type of food, levels of distance from transit stop).
+
+* Markers for every transit stop, colored by type of transit.
```

Comment:
There are _a lot_ of transit stops in the city. Might want to have a two-step process: select route, then show stops to select.